pkgs/by-name/ma/mapproxy/package.nix

```diff
@@ -1,28 +1,37 @@
-{ lib
-, python3
-, fetchPypi
+{
+  lib,
+  python3,
+  fetchFromGitHub,
 }:
 
 with python3.pkgs;
 buildPythonApplication rec {
   pname = "MapProxy";
```

Comment:
I suggest to rename pname name to 'mapproxy'.

Reply:
done